### ISSUE TITLE
new adapter + babel setup

### ISSUE THREAD

**Issue by jamesdixon** [aexmachina/factory-girl#104]
Hi,

I'm working on an adapter for [Objection.js]() and ran into the following error when attempting to run tests for it:

```
ObjectionAdapter #build builds the model:
     TypeError: Class constructor Model cannot be invoked without 'new'
      at new DummyObjectionModel (test/test-helper/DummyObjectionModel.js:32:145)
```

Looking at the Objection repo, [they claim it has something to do with the Babel setup](https://github.com/Vincit/objection.js/issues/388).

Do you have any thoughts on what would need to be changed to make things work with the current Factory Girl setup? I'm a complete newb with regards to Babel, so was hoping you might have some advice.

Thanks!

**Comment by oliviertassinari** [comment]
Yes, we have been facing this issue too. We had to hack the babel config, I can't remember how.

**Comment by jamesdixon** [comment]
@oliviertassinari any chance you could share your babel config? Would be greatly appreciated!
